.github/workflows/ghpages.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - name: Install and Build
         run: |
-          cd interactive
+          cd app
```

Comment:
Need to update L32, as well.

Reply:
Resolved in https://github.com/nmind/standards-checklist/pull/30/commits/2965d38617a5ee2b3545cd93170584dc43f5481b